webhook-connector/configure-the-webhook.md

```diff
@@ -4,50 +4,68 @@ description: Configurate the webhook in the Tradecloud portal
 
 # Configure the webhook
 
-## Configure your webhook in the Tradecloud1 portal
+## Configure your webhooks in the Tradecloud1 portal
 
-### Select Webhook
+### Select one or more webhooks
 
-As a company admin configure the webhook in your company settings on the [Tradecloud1 platform](http://portal.tradecloud1.com):
+As a company admin configure one or more webhooks in your company settings on the [Tradecloud1 platform](http://portal.tradecloud1.com):
 
-![](../.gitbook/assets/blue-sailboats-webhook.png)
+![Webhook Settings](../.gitbook/assets/webhook-settings.png)
 
-* Select **My team** in the menu below your avatar.
-* Select **Settings** in the page menu.
-* Select **Webhook** in the Integration section.
+* Select **My company** in the menu below your avatar.
+* The **Settings** tab in the Company menu should be selected.
+* Configure one or more **Webhook Integration**s below.
 
-{% hint style="info" %}
-The shipment webhook has a similar configuration section called "Shipments Webhook Integration".
+### Orders Webhook Integration
 
-The shipment webhook configuration will be moved into the integration section in the future.
-{% endhint %}
+Enable the **Order Webhook Integration** if you want to receive a webhook trigger when an order has been issued or changed:
+
+![Order Events](../.gitbook/assets/webhook-order-events.png)
+
+A default set of order events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
```

Comment:
"dependent on the capabilities of your integration and ERP system" -> And your process flow / functional requirements?

Reply:
Added ` and your order process requirements.`